src/locale/bootstrap-table-pt-PT.js

```diff
@@ -24,19 +24,19 @@ $.fn.bootstrapTable.locales['pt-PT'] = $.fn.bootstrapTable.locales['pt'] = {
     return `A mostrar ${pageFrom} at&eacute; ${pageTo} de ${totalRows} linhas`
   },
   formatSRPaginationPreText () {
-    return 'previous page'
+    return 'p&aacute;gina anterior'
```

Comment:
Why the html entites ?
`Á` is part of utf8 so i guess we dont need the html entities.

Reply:
saw a mix of html entities and utf8, and decided to stick with just one; do you prefer all in UTF8 instead?

Reply:
I dont see a reason to use html entites, without the code is more readable :-)

Reply:
updated! thanks for the quick feedback :)